meson.build

```diff
@@ -1433,19 +1433,19 @@ if vaapi_drm['use']
     features += vaapi_drm['name']
 endif
 
-vaapi_wayland = {
-    'name': 'vaapi-wayland',
-    'deps': dependency('libva-wayland', version: '>= 1.1.0', required: get_option('vaapi-wayland')),
+dmabuf_wayland = {
+    'name': 'dmabuf-wayland',
+    'deps': dependency('libva-wayland', version: '>= 1.1.0', required: get_option('dmabuf-wayland')),
 }
```

Comment:
Heh so this is where the naming gets confusing. There's a `vaapi-wayland` _library_ which is what this part of the meson build is looking for. That gets stored in `vaapi_wayland` _dictionary_, but this has nothing to do with the VO that we named `vo_vaapi_wayland`.

So basically, don't rename this particular part because it's actually is VAAPI. Just only rename `vo_dmabuf_wayland` internally in the VO (i.e. `video_out_dmabuf_wayland`). Same applies to waf of course.